okapi-core/src/main/java/org/folio/okapi/managers/InternalModule.java

```diff
@@ -256,11 +254,21 @@ public static ModuleDescriptor moduleDescriptor(String okapiVersion) {
         + "    \"permissionsRequired\" : [ \"okapi.proxy.tenants.upgrade.post\" ], "
         + "    \"type\" : \"internal\" "
         + "   }, {"
+        + "    \"methods\" :  [ \"GET\" ],"
+        + "    \"pathPattern\" : \"/_/proxy/tenants/{tenantId}/install\","
```

Comment:
Does this mean it's possible to check the status of a sync install?  Is it worth mentioning that in the docs?

Reply:
Yes. I can mention it.